templates/details/resources.html

```diff
@@ -16,312 +18,192 @@
-            {% if resource.type == "oci-image" %}
-            OCI Image
-            {% else %}
-            File
-            {% endif %}
-          </button>
-        </div>
-      </div>
-
-      <div tabindex="0" role="tabpanel" id="resource-image-tab" aria-labelledby="resource-image">
-        <ul class="p-detail-inline-list">
-          <li class="p-inline-list__item">
-            <i class="p-icon--revisions"></i> Revision: {{ resource.revision }}
-          </li>
-          <li class="p-inline-list__item">
-            <i class="p-icon--change-version"></i> Updated: -
-          </li>
-          <li class="p-inline-list__item has-panel js-panel-toggle">
-            {% if resource.type == "oci-image" %}
-            <i class="p-icon--docstrings"></i> Digest: {{ resource.short_digest }}
-            <div class="p-inline-list__panel">
-              {{ resource.digest }}
-            </div>
-            {% else %}
-            <i class="p-icon--docstrings"></i> Size: {{ resource.size }}
+    <h2 class="p-heading--4">
+      {{ resource.name }}
+      <span class="u-text--muted">
+        |
+        {% if resource.type == "oci-image" %}
+          OCI Image
+        {% else %}
+          File
+        {% endif %}
+      </span>
+    </h2>
+
+    <ul class="p-detail-inline-list">
+      <li class="p-inline-list__item">
+        <i class="p-icon--revisions" style="margin-right: 5px;"></i> Revision: {{ resource.revision }}
+      </li>
+      <li class="p-inline-list__item">
+        <i class="p-icon--change-version" style="margin-right: 5px;"></i> Updated: -
+      </li>
+      {% if resource.type == "oci-image" %}
+        <li class="p-inline-list__item has-panel js-panel-toggle">
+          <i class="p-icon--docstrings" style="margin-right: 5px;"></i>
+          Digest: {{ resource.short_digest }}
+          <div class="p-inline-list__panel">{{ resource.digest }}</div>
+        </li>
+      {% else %}
+      <li class="p-inline-list__item">
+        <i class="p-icon--docstrings" style="margin-right: 5px;"></i> Size: {{ resource.size }}
+        {% endif %}
+      </li>
+    </ul>
+
+    <pre><code>juju download --include-resources</code></pre>
+
+    <p>{{ resource.description }}</p>
+
+    <h3 class="p-heading--5">Release history</h3>
+
+    <table class="js-revisions-table">
+      <thead>
+        <tr>
+          <th>Revision</th>
+          <th>Date</th>
+          {% if resource.type == "file" %}
+            <th>Size (MB)</th>
+          {% endif %}
+        </tr>
+      </thead>
+      <tbody>
+        {% for revision in revisions[0:revisions_limit] %}
+          <tr>
+            <td>{{ revision.revision }}</td>
+            <td>-</td> <!-- update this in JS also once date field is available -->
+            {% if resource.type == "file" %}
+              <td>{{ revision.download.size * 1000 * 1000 }}</td>
```

Comment:
The API response is in bytes, it is not multiplying. But I am wondering if we can do what I have done for the size above this table. 

I used a library that automatically "humanizes" the right unit to show depending on the value.
![image](https://user-images.githubusercontent.com/6353928/125759800-a220dbd5-914f-4d6b-9be7-3294c996aa67.png)

Here is an example of the different outputs:
![image](https://user-images.githubusercontent.com/6353928/125760164-9882234f-bc3f-4e38-b7cc-c64917ca283f.png)

We can do the math by ourselves and show only MB but I think this is a better option.

@danielmutis do you think we can do this on each row and then remove the `(MB)` from the table header?

Reply:
@jkfran @steverydz I think the converting the sizes would be great. I wonder if it would always stay on MB or it would go up to GB? Because if it's not consistent then we can't keep it in the header only. Otherwise we can just display all sizes in MB.

Reply:
@danielmutis We can't know how big the resource could be or even how small. IMO it is not helpful to fix the unit to MB:
![image](https://user-images.githubusercontent.com/6353928/125763242-cecaebfd-f53d-4b77-9750-2b9833e880cb.png)

Reply:
@jkfran sorry I misunderstood your comment! yes, let's keep the unit in each row.